code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -901,26 +931,19 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 /datum/reagents/proc/is_full()
 	return total_volume >= maximum_volume
 
-/datum/reagents/proc/get_heatcapacity()
-	var/heat_capacity = 0
-
-	if(reagent_list.len)
-		for(var/datum/reagent/R in reagent_list)
-			heat_capacity += R.volume*R.specheatcap
-
-	return heat_capacity
-
 /datum/reagents/proc/get_overall_mass()
 	//M = DV
-
 	var/overall_mass = 0
-
-	if(reagent_list.len)
-		for(var/datum/reagent/R in reagent_list)
-			overall_mass += R.density*R.volume
-
+	for(var/datum/reagent/R in reagent_list)
+		overall_mass += R.density*R.volume
 	return overall_mass
 
+/datum/reagents/proc/get_thermal_mass()
```

Comment:
Why change the name away from the proper one?

Reply:
while the old "heat capacity" is technically still correct, i think thermal mass is much more unambiguous

Reply:
Heat capacity is the actual term with standard units. Though I guess this isn't in standard units anyway